app/src/modules/settings/index.ts

```diff
@@ -236,6 +239,29 @@ export default defineModule({
 				},
 			],
 		},
+		{
+			path: 'market',
```

Comment:
I'd rather prefer calling the route `marketplace` for consistency, amongst other usages of `market` vs. `marketplace`.